site/src/examples/tag/Categories.tsx

```diff
@@ -0,0 +1,28 @@
+import { ReactElement } from "react";
+import { Tag } from "@salt-ds/lab";
+import { GridLayout } from "@salt-ds/core";
+
+export const Categories = (): ReactElement => (
```

Comment:
My personal angle, would be nice to see both/three variants for all these categories.

Reply:
@origami-z  will check with Shey once she is back from OOO and follow up in another PR since it is not a blocker